File: app/src/main/java/com/example/mettest/models/PinkItems.java

package com.example.mettest.models;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class PinkItems {

    @SerializedName("status")
    @Expose
    public Status status;
    @SerializedName("result")
    @Expose
    public Result result;

}

File: app/src/main/java/com/example/mettest/models/PinkItems.kt
package com.example.mettest.models

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class PinkItems {
    @SerializedName("status")
    @Expose
    var status: Status? = null

    @SerializedName("result")
    @Expose
    var result: Result? = null
}